app/src/main/java/com/sapuseven/untis/data/databases/UserDatabase.kt

```diff
@@ -73,6 +73,13 @@ class UserDatabase private constructor(context: Context) : SQLiteOpenHelper(cont
 					db.execSQL("INSERT INTO ${UserDatabaseContract.Users.TABLE_NAME} SELECT * FROM ${UserDatabaseContract.Users.TABLE_NAME}_v3;")
 					db.execSQL("DROP TABLE ${UserDatabaseContract.Users.TABLE_NAME}_v3")
 				}
+				4 -> {
+					db.execSQL("ALTER TABLE ${UserDatabaseContract.Users.TABLE_NAME} RENAME TO ${UserDatabaseContract.Users.TABLE_NAME}_v4")
+					db.execSQL(UserDatabaseContract.Users.SQL_CREATE_ENTRIES_V5)
+					db.execSQL("ALTER TABLE ${UserDatabaseContract.Users.TABLE_NAME}_v4 ADD COLUMN ${UserDatabaseContract.Users.COLUMN_NAME_PROFILENAME}")
```

Comment:
Do you really need this statement?

Reply:
I don't, that was a remainder from a previous commit. Thank you!